android/src/main/java/com/rudderstack/android/ConfigurationAndroid.kt

```diff
@@ -99,21 +156,22 @@ interface ConfigurationAndroid : Configuration {
             isPeriodicFlushEnabled: Boolean = Defaults.IS_PERIODIC_FLUSH_ENABLED,
             autoCollectAdvertId: Boolean = Defaults.AUTO_COLLECT_ADVERT_ID,
             multiProcessEnabled: Boolean = Defaults.MULTI_PROCESS_ENABLED,
-            defaultProcessName: String?= Defaults.DEFAULT_PROCESS_NAME,
+            defaultProcessName: String? = Defaults.DEFAULT_PROCESS_NAME,
             advertisingId: String? = null,
             deviceToken: String? = null,
-            logger: Logger = AndroidLogger,
+            logLevel: Logger.LogLevel = Logger.LogLevel.NONE,
```

Comment:
you don't need the log level here.

Reply:
Did you mean to remove the default logLevel value? Should I keep it like this: `logLevel: Logger.LogLevel,`? Or something else?

Reply:
He means you can directly set this in the ` logger: Logger = AndroidLogger(DEFAULT_LOG_LEVEL),` constructor parameter by setting a default `DEFAULT_LOG_LEVEL` parameter.